### PR TITLE
Acl functionality for routes

### PR DESCRIPTION
**Pull request by NikolaL87**

### CODE REVIEW

File: www/front_src/src/App.js [hunk 1]
```diff
@@ -28,11 +32,35 @@ class App extends Component {
...
+      .then(({data}) => this.setState({acls: data, aclsLoaded: true}))
+  }
+
+  linkRoutesAndComponents = () => {
+    const {acls} = this.state;
+    return routes.map(({ path, comp, ...rest }) => (
+      <ClassicRoute
+        history={history}
+        path={path}
+        component={acls.indexOf(`/${path.split('/_CENTREON_PATH_PLACEHOLDER_/')[1]}`) > -1 ? NotAllowedPage : comp}
```
Comment (kduret) [inline www/front_src/src/App.js]: you could use includes ;)
https://developer.mozilla.org/fr/docs/Web/JavaScript/Reference/Objets_globaux/Array/includes

File: www/front_src/src/App.js [hunk 2]
```diff
@@ -28,11 +32,35 @@ class App extends Component {
     this.setState({ isFullscreenEnabled: true });
   }
 
+  UNSAFE_componentWillMount = () => {
+    axios("internal.php?object=centreon_acl_webservice&action=getCurrentAcl")
```
Comment (kduret) [inline www/front_src/src/App.js]: where is this webservice ?
should be "currentAcl" instead of "getCurrentAcl"

Comment (NikolaL87) [reply www/front_src/src/App.js]: I have this webservice. @victorvassilev send me that one yesterday and it works :)

File: www/front_src/src/App.js (hunk 1 above)
Comment (NikolaL87) [reply www/front_src/src/App.js]: Old school :) I think that I saw somewhere that this is the faster :) I can change if you want @kduret ? :)

File: www/front_src/src/App.js (hunk 2 above)
Comment (kduret) [reply www/front_src/src/App.js]: ok so commit it in this branch :)
cause if we merge your PR, the application will be broken

**Pull request merged**